.github/workflows/tests.yaml

```diff
@@ -86,11 +86,11 @@ jobs:
             - name: Run Tests
               timeout-minutes: 20
               run: |
-                  scripts/tests/test_suites.sh
+                  scripts/tests/test_suites.sh -n
```

Comment:
These changes are needed when using the minmdns unicast behavior?  Or just extras on the side?

Reply:
Needed with minmdns unicast behaviour. What is happening:

- Darwin works because it uses bonjour (once central mdns)
- Avahi does not work and I failed to properly debug it (cannot get avahi to run reliably on my workstation due to corp restrictions)

so in minmdns, minmdns was built as 'the only small mdns server' so it does not share. We start both chip-tool and all-clusters with "minmdns" and both listen on 5353. Broadcast works because all interfaces get the packet, but switching to unicast makes only one of them receiving packets and that fails (all-clusters-app responds to itself).

I will update the bug summary once I finalize the patch. I have been fighting with netns all day :(

Reply:
Updated summary (in a slightly more brief manner)